config_panel.toml

```diff
@@ -0,0 +1,209 @@
+        help = "Defaults to: '10M' . If you are using a reverse proxy you may also need to set this value in your reverse proxy's config. Notably Nginx has a small max body size by default. See https://matrix-org.github.io/synapse/latest/reverse_proxy.html."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.web_client_location]
+        ask = "URL to the web client which / will redirect to."
+        type = "url"
+        help = "Basically Element instance the server should redirect to."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.client_base_url]
+        ask = "URL for client links within the email notifications."
+        type = "url"
+        help = "Used to be called 'riot_base_url', still supported"
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.invite_client_location]
+        ask = "Web client location to direct users to during an invite."
+        type = "url"
+        help = "This is passed to the identity server as the org.matrix.web_client_location key. Defaults to unset, giving no guidance to the identity server."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.enable_group_creation]
+        ask = "Allow non-server-admin Users to create Spaces?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'false'."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.e2e_enabled_by_default]
+        ask = "End-to-End Encryption by default for locally-created Rooms"
+        type = "select"
+        choices = ["all", "invite", "off"]
+        help = '''
+        * "all": any locally-created room
+        * "invite": any room created with the "private_chat" or "trusted_private_chat" room creation presets
+        * "off": this option will take no effect
+        '''
+        bind = "encryption_enabled_by_default_for_room_type:/etc/matrix-__APP__/homeserver.yaml"
+
+    [client.welcome]
+    name = "Welcome User Experience"
+
+        [client.welcome.auto_join_rooms]
+        ask = "Auto Join new Users in following Rooms."
+        type = "string"
+        help = "Room should be public. If not existing, will be created as public and federated by default."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.welcome.autocreate_auto_join_rooms]
+        ask = "Create room for Auto Join if not existing."
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Will be created as public and federated by default. Can be customised with the settings auto_join_*."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.welcome.auto_join_rooms_for_guests]
+        ask = "Enable Auto Join Room for Guests?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'true'."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
```

Comment:
Same here it would be good to have a full description.

Reply:
done